app/services/redis/client.py

```
@@ -290,6 +290,131 @@ async def expire(self, key: str, seconds: int) -> bool:
+        except RedisError as e:
+            logger.error(f"Redis SISMEMBER error for key {key}: {e}")
+            return False
+
+    # ─────────────────────────────────────────────────────────────────
+    # HASH OPERATIONS (for pod info storage)
+    # ─────────────────────────────────────────────────────────────────
+
+    async def hset(
+        self,
+        key: str,
+        field: str = None,
```

Ruff (0.14.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)